staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go

```diff
@@ -368,6 +368,12 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 			}
 			gvToTypes[gv] = append(gvToTypes[gv], t)
 		}
+		if len(gvToTypes[gv]) == 0 {
```

Comment:
Can't we skip somehow around line 400 with a simple continue? All these deletes belong look hacky.

Reply:
sorry, devel freeze rush + PTO :-)

No, even `generator.Packages(customArgs, ...)` and  `packageForScheme(customArgs, ...)`  below use `customArgs.Groups`, potentially with groups that should be skipped.